kibble/models/film.go

```diff
@@ -110,23 +107,26 @@ func (films *FilmCollection) MakeTitleSlugsUnique() {
 	groups := make(map[string][]int, 0)
 
 	// create a grouping of slugs to films first
-	for fi, film := range *films {
+	cnt := 0
```

Comment:
Go maps aren't indexed... does this do what you are expecting it to?

Reply:
Removed. I can't remember what I was trying to do haha.